cypress/e2e/integration/MainPage.cy.ts

```diff
@@ -0,0 +1,95 @@
+import { DateProps } from "interfaces/date_props.interface";
+
+describe("Renders Main page of app", () => {
+  const lengthOfItems = 5;
+  it("Renders correctly", () => {
+    cy.visit("/");
+  });
+
+  it("Testing date", () => {
```

Comment:
Need more details about the test

Reply:
This test just getting the current date and checking that details are showing on the screen are the same like the object of the date.

Reply:
But the title of the test needs to be clear to understand what the test does without seeing the code